docs/spec/overview.md

```diff
@@ -0,0 +1,80 @@
+# Resource Groups
```

Comment:
Feel free to steal from https://github.com/knative/docs/blob/master/eventing/README.md#architecture if it makes sense.

Reply:
I linked to it. That doc has the wrong abstraction. We did not abstract the bus, we abstracted the channel.

Reply:
Or we have the wrong API group names. I could be convinced either way.